src/plugins/telemetryTable/components/table.vue

```diff
@@ -494,23 +501,24 @@ export default {
     this.table.tableRows.on('sort', this.updateVisibleRows);
     this.table.tableRows.on('filter', this.updateVisibleRows);
 
+    this.table.configuration.on('change', this.updateConfiguration);
```

Comment:
Why did this move?

Reply:
I think I may have just moved it to be with the rest of the listeners. I can move it back if you think we should.